recipes/service.rb

```diff
@@ -31,8 +31,21 @@
   when 'arch', 'exherbo'
     service_name 'systemd-sysctl'
     provider Chef::Provider::Service::Systemd
+  when 'centos','redhat'
```

Comment:
Needs a space after the comma.

Also, this fails [FC024](https://acrmp.github.io/foodcritic/#FC024).